README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/wso2-ballerina/module-amazons3.svg?branch=master)](https://travis-ci.org/wso2-ballerina/module-amazons3)
+[![Build Status](https://travis-ci.org/ballerina-platform/module-aws.s3.svg?branch=master)](https://travis-ci.org/ballerina-platform/module-aws.s3)
```

Comment:
Here we no need to change the module name
```suggestion
[![Build Status](https://travis-ci.org/ballerina-platform/module-amazons3.svg?branch=master)](https://travis-ci.org/ballerina-platform/module-amazons3)
```

Reply:
Fixed.